Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -116,6 +116,9 @@ from one cluster to another:
   kubectl --context=$CLUSTER1 get secret -n kube-system cilium-ca -o yaml | \
     kubectl --context $CLUSTER2 create -f -
 
+  # Annotate the secret so that it isn't removed by Helm
```

Comment:
```suggestion
  ## Annotate the secret so that it isn't removed by Helm
```

I think Netlify complains about the above -- the `#` is read as a prompt symbol but it fails to parse the line as a prompt (because of the opening `'` I suppose) and complains it cannot apply the relevant style to the code block.